checkov/common/output/gitlab_sast.py

```diff
@@ -0,0 +1,159 @@
+from __future__ import annotations
+
+from datetime import datetime
+from typing import TYPE_CHECKING, Any
+from uuid import uuid4
+
+from checkov.common.output.cyclonedx_consts import SCA_CHECKTYPES
+from checkov.version import version
+
+if TYPE_CHECKING:
+    from checkov.common.output.record import Record
+    from checkov.common.output.report import Report
+
+DEFAULT_SEVERITY_GITLAB_LEVEL = "Unknown"
+SEVERITY_TO_GITLAB_LEVEL = {
+    "critical": "Critical",
+    "high": "High",
+    "medium": "Medium",
+    "low": "Low",
+    "none": "Info",
+}
+
+
+class GitLabSast:
+    def __init__(self, reports: list[Report]):
+        self.reports = reports
+
+        self.sast_json = self.create_sast_json()
+
+    def create_sast_json(self) -> dict[str, Any]:
+        return {
+            "schema": "https://gitlab.com/gitlab-org/security-products/security-report-schemas/-/raw/v15.0.4/dist/sast-report-format.json",
+            "version": "15.0.4",
+            "scan": self._create_scan(),
+            "vulnerabilities": self._create_vulnerabilities(),
+        }
+
+    def _create_scan(self) -> dict[str, Any]:
+        current_datetime = datetime.utcnow().strftime("%Y-%m-%dT%H:%M:%S")
+        scanner = {
+            "id": "checkov",
+            "name": "Checkov",
+            "url": "https://www.checkov.io/",
+            "vendor": {
+                "name": "Bridgecrew",
+            },
+            "version": version,
+        }
+
+        return {
+            "start_time": current_datetime,  # needs to be done properly in a later stage
+            "end_time": current_datetime,
```

Comment:
what's the point in adding `ent_time` if it's the same? perhaps use a placeholder?

Reply:
`end_time` is a required field by the JSON schema of GitLab SAST